packages/docs-app/src/examples/core-examples/menuItemExample.tsx

```diff
@@ -33,33 +34,34 @@ export function MenuItemExample(props: ExampleProps) {
     const [submenuEnabled, setSubmenuEnabled] = React.useState(true);
     const [roleStructure, setRoleStructure] = React.useState<MenuItemProps["roleStructure"]>("menuitem");
 
-    const isSelectedOptionAvailable = roleStructure === "listoption" && !iconEnabled;
+    const isSelectable = roleStructure === "listoption";
 
     const options = (
         <>
             <H5>Menu props</H5>
-            <Switch label="Large" checked={large} onChange={handleBooleanChange(setLarge)} />
+            <SizeSelect size={size} onChange={setSize} />
             <H5>MenuItem props</H5>
             <Switch label="Active" checked={active} onChange={handleBooleanChange(setActive)} />
             <Switch label="Disabled" checked={disabled} onChange={handleBooleanChange(setDisabled)} />
             <PropCodeTooltip
                 content={
-                    isSelectedOptionAvailable ? undefined : (
+                    isSelectable ? undefined : (
                         <>
                             <Code>selected</Code> prop has no effect when <br />
-                            <Code>roleStructure="menuitem"</Code> or when an icon is set
+                            <Code>roleStructure="menuitem"</Code>
                         </>
                     )
                 }
+                disabled={isSelectable}
             >
                 <BooleanOrUndefinedSelect
-                    disabled={!isSelectedOptionAvailable}
+                    disabled={!isSelectable}
                     label="Selected"
                     value={selected}
                     onChange={setSelected}
                 />
             </PropCodeTooltip>
-            <Switch label="Enable icon" checked={iconEnabled} onChange={handleBooleanChange(setIconEnabled)} />
+            <Switch label="Show custom icon" checked={iconEnabled} onChange={handleBooleanChange(setIconEnabled)} />
```

Comment:
"Custom icon" may not be the right terminology here. It creates ambiguity with the custom SVG icon shown elsewhere in this docs page.